pkgs/development/python-modules/torch/default.nix

```diff
@@ -203,7 +203,7 @@ let
     "MPI cudatoolkit does not match cudaPackages.cudatoolkit" =
       MPISupport && cudaSupport && (mpi.cudatoolkit != cudaPackages.cudatoolkit);
     "Magma cudaPackages does not match cudaPackages" =
-      cudaSupport && (effectiveMagma.cudaPackages != cudaPackages);
+      cudaSupport && (effectiveMagma.cudaPackages.cudaVersion != cudaPackages.cudaVersion);
```

Comment:
```suggestion
    # This used to be a deep package set comparison, but it made it evaluation too strict. This was bad e.g. for cudaPackages' `aliases.nix`.
    "Magma cudaPackages does not match cudaPackages. To apply overrides consistently, use overlays" =
      cudaSupport && (effectiveMagma.cudaPackages.cudaVersion != cudaPackages.cudaVersion);
```

Reply:
Done this, but tweaked the wording a bit.